x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSetup } from 'kibana/server';
+import { CustomIntegrationsPluginSetup } from '../../../../src/plugins/custom_integrations/server';
+import { APP_ID } from '../common/constants';
+
+export function registerIntegrations(
+  core: CoreSetup,
+  customIntegrations: CustomIntegrationsPluginSetup
+) {
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_with_gdal',
+    title: i18n.translate('xpack.maps.registerIntegrations.gdal.integrationTitle', {
+      defaultMessage: 'Upload geo data with GDAL',
+    }),
+    description: i18n.translate('xpack.maps.registerIntegrations.gdal.integrationDescription', {
+      defaultMessage:
+        'Upload shapefiles or ingest from relational databases such as PostGIS and OracleSpatial',
+    }),
+    uiInternalPath:
+      'https://www.elastic.co/blog/how-to-ingest-geospatial-data-into-elasticsearch-with-gdal',
```

Comment:
Can we link to something inside the product instead?

Reply:
Not yet. We only have the blog post for now. It is not uncommon for the data-integrations page to link to blog posts, e.g. the Fleet announcements

I do agree that in-app documentation would be helpful.